test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -203,5 +203,39 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
         }
     }
 
+    /**
+     * Helper function to set user permissions. The function works by deleting
+     * all current permissions for the user then adding the specified permissions
+     * to the TestUser.
+     *
+     * @param array $permissions List of permissions codes to add TestUser
+     *
+     * @return none
+     */
+    function setupPermissions($permissions) {
+        $this->DB->delete("user_perm_rel", array("UserID" => '999990'));
+        foreach ($permissions as $value) {
+            $prepare = $this->DB->prepare(
```

Comment:
This should be prepared outside of the loop, otherwise what's the point of preparing it instead of just using the insert wrapper?